### PR TITLE
OCPBUGS-24114: IBMCloud: Update ibm-vpc-block-csi-driver images to be consistent with ART

### PR DESCRIPTION
**Pull request by cjschaef**
Update the YAML to remove image overwrites from the release config, so the ibm-vpc-block-csi-driver and
ibm-vpc-block-csi-driver-operator repos can set images.

Related: https://issues.redhat.com/browse/OCPBUGS-24114
Related: https://github.com/openshift/machine-api-provider-ibmcloud/pull/27#issuecomment-1841586140
Related: https://github.com/openshift/release/pull/46309

### CODE REVIEW

**Comment by cjschaef** [comment]
/pj-rehearse more pull-ci-openshift-ibm-vpc-block-csi-driver-master-verify-commits pull-ci-openshift-ibm-vpc-block-csi-driver-master-verify-deps pull-ci-openshift-ibm-vpc-block-csi-driver-operator-master-e2e-ibmcloud-csi pull-ci-openshift-ibm-vpc-block-csi-driver-operator-master-e2e-ibmcloud-csi-extended pull-ci-openshift-ibm-vpc-block-csi-driver-operator-master-images pull-ci-openshift-ibm-vpc-block-csi-driver-operator-master-unit pull-ci-openshift-ibm-vpc-block-csi-driver-operator-master-verify pull-ci-openshift-ibm-vpc-block-csi-driver-operator-master-verify-deps

File: ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml [hunk 1]
```diff
@@ -25,9 +21,6 @@ images:
     ocp_4.15_base-rhel9:
       as:
       - registry.ci.openshift.org/ocp/4.15:base-rhel9
-    ocp_builder_rhel-9-golang-1.20-openshift-4.15:
-      as:
-      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.15
```
Comment (dobsonj) [inline ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml:30]: Can you remove this whole `inputs` section?
Basically revert the changes the bot made for these files in https://github.com/openshift/release/commit/fa388b204f7d2282d36805de37058216f0336621
```
--- a/ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml
+++ b/ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml
@@ -3,6 +3,14 @@ base_images:
     name: commitchecker
     namespace: ci
     tag: "4.15"
+  ocp_4.15_base-rhel9:
+    name: "4.15"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.20-openshift-4.15:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.15
   tests-private:
     name: tests-private
     namespace: ci
@@ -13,6 +21,13 @@ build_root:
 canonical_go_repository: sigs.k8s.io/ibm-vpc-block-csi-driver
 images:
 - dockerfile_path: Dockerfile.openshift
+  inputs:
+    ocp_4.15_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.15:base-rhel9
+    ocp_builder_rhel-9-golang-1.20-openshift-4.15:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.15
   to: ibm-vpc-block-csi-driver
 promotion:
   name: "4.15"
```

Comment (dobsonj) [reply ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml:30]: Same for the operator as well

Comment (cjschaef) [reply ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml:30]: Sure, I can do that.

Comment (cjschaef) [reply ci-operator/config/openshift/ibm-vpc-block-csi-driver/openshift-ibm-vpc-block-csi-driver-master.yaml]: I think I got them updated that way.

**Comment by cjschaef** [comment]
Still see the same errors as before for `csi-driver`
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/46436/rehearse-46436-pull-ci-openshift-ibm-vpc-block-csi-driver-operator-master-e2e-ibmcloud-csi/1732426999365898240/artifacts/e2e-ibmcloud-csi/gather-extra/artifacts/pods/openshift-cluster-csi-drivers_ibm-vpc-block-csi-node-9d5bg_csi-driver.log
```
/bin/ibm-vpc-block-csi-driver: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by /bin/ibm-vpc-block-csi-driver)
/bin/ibm-vpc-block-csi-driver: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by /bin/ibm-vpc-block-csi-driver)
```

I take it this patch isn't sufficient.
